docker/vnc/fonts/calcutta/Calcutta-Medium.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml">
+
+<head>
+	<meta content="text/html; charset=utf-8" http-equiv="content-type" />
+	<meta content="en-us" http-equiv="Content-Language" />
+	<title>Calcutta Medium Regular - Web Font Specimen</title>
+	<link rel="stylesheet" type="text/css" media="screen" href="Calcutta-Medium.css" />
+	<style type="text/css" media="screen">
+		body {
+			font-size: 14px;
+			font-family: "Calcutta-Medium", Georgia;
+		}
+	</style>
+</head>
+
+<body>
+	<p contenteditable="true">The quick brown fox jumps over the lazy dog. $123.45!</p>
+</body>
+
+</html>
```

Comment:
Do you want to delete this now that it has been tested?

Reply:
Yep let's delete it.